2996_reflection/reflection.md

```diff
@@ -4025,6 +4013,26 @@ Adjust paragraph 3 of [temp.arg.general] to not apply to splice template argumen
 
 [3]{.pnum} [A `$template-argument$` of the form `$splice-specifier$` is interpreted as a `$splice-template-argument$`.]{.addu} In a `$template-argument$` [that is not a `$splice-template-argument$`]{.addu}, an ambiguity between a `$type-id$` and an expression is resolved to a `$type-id$`, regardless of the form of the corresponding `$template-parameter$`.
 
+::: example2
+```cpp
+template<class T> void f();
+template<int I> void f();
+
+void g() {
+  f<int()>();       // int() is a type-id: call the first f()
+```
+::: addu
+```cpp
+  constexpr int x = 42;
+  f<[:^int:]>();      // splice-template-argument: calls the first f()
+  f<[:^x:]>();      // splice-template-argument: calls the second f()
+```
+:::
+```cpp
+}
+```
```

Comment:
Er? 

You're just adding lines to an existing example right? That's a `diff` code block. 
```
```diff
  template<class T> void f();
  template<int I> void f();

  void g() {
    f<int()>();       // int() is a type-id: call the first f()

+  constexpr int x = 42;
+  f<[:^int:]>();      // splice-template-argument: calls the first f()
+  f<[:^x:]>();      // splice-template-argument: calls the second f()
  }
```